deploy/deploy.ts

```diff
@@ -99,12 +99,23 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       syntheticAggregator.address,
       tokenSpender.address,
     );
-  await registryInstance.connect(governor).setProtocolExecutionReserveClaimer(deployer.address);
-  await registryInstance.connect(governor).setProtocolRedemptionReserveClaimer(redemptionReserveClaimer.address);
-  await registryInstance.connect(governor).addToWhitelist(core.address);
+  await tx1.wait();
+
+  const tx2 = await registryInstance.connect(governor).setProtocolExecutionReserveClaimer(deployer.address);
+  await tx2.wait();
+
+  const tx3 = await registryInstance
+    .connect(governor)
+    .setProtocolRedemptionReserveClaimer(redemptionReserveClaimer.address);
+  await tx3.wait();
+
+  const tx4 = await registryInstance.connect(governor).addToWhitelist(core.address);
+  await tx4.wait();
 
   const coreInstance = <Core>await ethers.getContract("Core");
-  await coreInstance.connect(governor).updateProtocolAddresses();
+
+  const tx5 = await coreInstance.connect(governor).updateProtocolAddresses();
+  await tx5.wait();
 };
 
 export default func;
```

Comment:
- Add func.ID to identify deployment script
- Add "return true" at the end of the deployment